geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/BinnedArray.scala

```diff
@@ -25,7 +26,7 @@ import scala.reflect.ClassTag
   * @param bounds upper and lower bounds for the input values
   * @tparam T type of input value
   */
-abstract class BinnedArray[T](val length: Int, val bounds: (T, T)) {
+abstract class BinnedArray[T](val length: Int, val bounds: (T, T)) extends LazyLogging {
```

Comment:
Ah, maybe the logger is designed to have errors go to the log file instead of stdout?